src/snapshot-io/types.ts

```diff
@@ -1,46 +1,28 @@
-export enum VoteChoice {
-  Yes,
-  No,
-}
-
-export interface zDAO {
-  id: string; // Global zDAO identifier
-  zNA: string; // Linked zNA
-  title?: string; // zDAO title
-  creator: string; // Creator wallet address
-  owner: string; // Owner wallet address
-  avatar?: string; // Avatar uri (https link)
-  network: string; // Chain id
-  safeAddress: string; // Gnosis Safe address
-  votingToken: string; // Voting token address
-}
-
-export interface TokenMetaData {
-  recipient: string; // asset recipient address
-  token: string; // asset token address
-  amount: string; // BigNumber string mutiplied by decimals
-}
-
-export interface Proposal {
+export interface SnapshotProposal {
   id: string; // proposal id
-  author: string;
-  title: string;
-  body?: string;
-  ipfs: string;
-  choices: VoteChoice[];
+  type: string; // proposal type (e.g. single-choice)
+  author: string; // proposal creator
+  title: string; // proposal title
+  body?: string; // empty body if not defined
+  ipfs: string; // uri to ipfs which contains proposal information and signature
+  choices: string[];
   created: Date;
-  started: Date;
-  ended: Date;
-  status: 'pending' | 'active' | 'closed';
-  network: string;
-  snapshot: string;
-  scores: number[];
-  metadata: TokenMetaData & { sender: string };
-  zDAOId: string;
+  start: Date;
+  end: Date;
+  state: 'pending' | 'active' | 'closed';
+  network: string; // chain id
+  snapshot: string; // snapshot block number
+  scores: number[]; // scores per all the choices
+  votes: number; // number of voters
+}
+
+export interface ProposalResult {
+  resultsByVoteBalance: number[];
+  sumOfResultsBalance: number;
 }
 
-export interface Vote {
+export interface SnapshotVote {
   voter: string;
-  choice: VoteChoice;
+  choice: number[]; // VoteChoice;
```

Comment:
Reason to use number instead? If it's required (since it's a type conversion) you could add a comment explaining the conversion and relation to `VoteChoice`